### ISSUE TITLE
seadas to install ocssw ,have the problem,connection refused.

### ISSUE THREAD

**Issue by funny000** [seadas/seadas#95]
hello,Do you solves the problem ?,I ,now ,installation the OCSSW ,but have the 
problem--please check network connection or 'seadas.ocssw.root'variable in the "seadas.config" file,possible cause of error:Connection refused

**Comment by dlmu2951** [comment]
yeah, domestic network condition may not support this ways to install OCSSW. Our school-network can fetch the repository of nasa oceancolor and the processing modules have been downloaded successfully, however, only version 7.5 can be used, some problems of new version of seadas may arise when the produre is working even if the network is available for download. 
So i advise u to download seadas7.5 in your ubuntu or vm, i can send u the OCSSW file package which may work in your computer.
http://blog.sina.com.cn/s/blog_942ff76e0102v3n6.html. THIS may help you to reach the goal

**Comment by funny000** [comment]
Thank your reply. My computer has install the ubuntu-linux and the version 7.5.3 of seadas. Same version as seadas7.5, Do you send me the OCSSW file package?




------------------ 原始邮件 ------------------
发件人: "dlmu2951"<notifications@github.com>;
发送时间: 2019年5月16日(星期四) 下午5:10
收件人: "seadas/seadas"<seadas@noreply.github.com>;
抄送: "Funny"<2152550864@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [seadas/seadas] seadas to install ocssw ,have theproblem,connection refused. (#95)




yeah, domestic network condition may not support this ways to install OCSSW. Our school-network can fetch the repository of nasa oceancolor and the processing modules have been downloaded successfully, however, only version 7.5 can be used, some problems of new version of seadas may arise when the produre is working even if the network is available for download.
 So i advise u to download seadas7.5 in your ubuntu or vm, i can send u the OCSSW file package which may work in your computer.
 http://blog.sina.com.cn/s/blog_942ff76e0102v3n6.html. THIS may help you to reach the goal

**Comment by dlmu2951** [comment]
>e
There may be some errors, but you can try it. Leave your email and i send you the file package
If possible, we can communicate with each other in using seadas

**Comment by funny000** [comment]
Thank you, my email is funny777@foxmail.com.


------------------ 原始邮件 ------------------
发件人: "dlmu2951"<notifications@github.com>;
发送时间: 2019年5月17日(星期五) 中午1:50
收件人: "seadas/seadas"<seadas@noreply.github.com>;
抄送: "Funny"<2152550864@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [seadas/seadas] seadas to install ocssw ,have theproblem,connection refused. (#95)



 
e
 There may be some errors, but you can try it. Leave your email and i send you the file package
 If possible, we can communicate with each other in using seadas

**Comment by jian940** [comment]
hello,do you solve the problem about installing the OCSSW file package?Now,i try to study the seadas,but i can not get the OCSSW file package .Can you share your OCSSW file package? thank you very much, my email is 18810916182@163.com

**Comment by dlmu2951** [comment]
> hello,do you solve the problem about installing the OCSSW file package?Now,i try to study the seadas,but i can not get the OCSSW file package .Can you share your OCSSW file package? thank you very much, my email is [18810916182@163.com](mailto:18810916182@163.com)

have u ever installed the file package successfully？OCSSW is so large, i can send u the file by Teamviewer or other

**Comment by funny000** [comment]
Hello，Do you have install the cossw?
